api/RTCIceTransport.json

```diff
@@ -356,6 +408,58 @@
           }
         }
       },
+      "selectedcandidatepairchange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceTransport/selectedcandidatepairchange_event",
+          "description": "<code>selectedcandidatepairchange</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
This [landed in Chrome 71](https://storage.googleapis.com/chromium-find-releases-static/ba8.html#ba8f43f2308d6186e8b164fb4d0a1f9e8dce53aa).